components/widgets/resources/src/main/resources/resources/uicomponents/widgets/extraGeneVariantData.js

```diff
@@ -353,6 +352,14 @@ var ExtraGeneVariantData = (function (ExtraGeneVariantData) {
       newMoreInfoRow.insert(varMoreInfoInner);
       variantFooter.insert({before : newMoreInfoRow});
 
+      this.restrictNumericInput(newMoreInfoRow);
+
+      // get the last value of reference_genome of all variantsto create a new variant row with this value
+      refGenomes = this._geneTable.select('select:[id$=_reference_genome]');
+      if (refGenomes && refGenomes.length > 0) {
```

Comment:
`select` is guaranteed to return an array, no need to check if it's define. Use only the second part of the condition.